server/sonar-db-dao/src/main/java/org/sonar/db/ce/CeQueueDao.java

```diff
@@ -84,6 +85,10 @@ public int countByQuery(DbSession dbSession, CeTaskQuery query) {
     return mapper(dbSession).selectPendingByMinimumExecutionCount(minExecutionCount);
   }
 
+  public void resetTasksWithUnknownWorkerUUIDs(DbSession dbSession, Set<String> knownWorkerUUIDs) {
+    mapper(dbSession).resetTasksWithUnknownWorkerUUIDs(knownWorkerUUIDs, system2.now());
```

Comment:
as the method is taking a collection as a parameter which is used to populate a "IN" clause, `org.sonar.db.DatabaseUtils#executeLargeUpdates` should be used